### PR TITLE
Update l10n documentation to remove Verbatim

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: locale/README.txt [hunk 1]
```diff
@@ -2,18 +2,12 @@ We are using English strings for gettext message ids.
...
-Assuming you want to add 'fr':
-1) mkdir -p locale/fr/LC_MESSAGES
-2) ./manage.py merge
-
-or
-1) msginit --no-translator -l fr -i templates/LC_MESSAGES/django.pot -o fr/LC_MESSAGES/django.po
-2) repeat for other POT files
+Assuming you want to add the locale 'xx':
+1) mkdir -p locale/xx/LC_MESSAGES
+2) for pot in locale/templates/LC_MESSAGES/* ; do msginit --no-translator -l xx -i $pot -o locale/xx/LC_MESSAGES/`basename -s .pot $pot`.po ; done
```
Comment (jezdez) [inline locale/README.txt]: Let's add that to a new make task, e.g. `make langauge xx`.

**Comment by jezdez** [comment]
rwc+

File: locale/README.txt (hunk 1 above)
Comment (robhudson) [inline locale/README.txt]: Didn't think of that. :+1:

**Comment by robhudson** [comment]
Updated if you want to review my Makefile skills. :)

File: Makefile [hunk 2]
```diff
@@ -43,5 +43,11 @@ in_vagrant:
 on_host:
 	@if [ ${IN_VAGRANT} -eq 1 ]; then echo "*** Run on host ***"; exit 1; fi
 
+locale:
+	mkdir -p locale/$(LOCALE)/LC_MESSAGES && \
```
Comment (jezdez) [inline Makefile]: Can you add a `@` in front of the mkdir call to make make not print the command before running it?

**Comment by jezdez** [comment]
rwc+

**Pull request merged**